libraries/chain/wasm_interface.cpp

```diff
@@ -46,6 +48,42 @@ namespace eosio { namespace chain {
       U32 _num_bytes;
    };
 
+   // account.h/hpp expected account API balance interchange format
+   // must match account.hpp account_balance definition
+   PACKED_STRUCT(
+   struct account_balance
+   {
+      /**
+      * Name of the account who's balance this is
+      * @brief Name of the account who's balance this is
+      */
+      account_name account;
+
+      /**
+      * Balance for this account
+      * @brief Balance for this account
+      */
+      asset eos_balance;
+
+      /**
+      * Staked balance for this account
+      * @brief Staked balance for this account
+      */
+      asset staked_balance;
+
+      /**
+      * Unstaking balance for this account
+      * @brief Unstaking balance for this account
+      */
+      asset unstaking_balance;
+
+      /**
+      * Time at which last unstaking occurred for this account
+      * @brief Time at which last unstaking occurred for this account
+      */
+      time last_unstaking_time;
+   })
```

Comment:
No ; ? Maybe the macro adds it.

Reply:
#define PACKED_STRUCT(definition) definition __attribute__((packed));